assignments/function-conversion.js

```diff
@@ -1,30 +1,56 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
+// ============== 1 ====================
 // let myFunction = function () {
 // console.log("Function was invoked!");
 // };
 // myFunction();
 
+let myFunction = _ => console.log("Function was invoked!");
```

Comment:
If you are not passing a parameter, use empty parentheses. 
```suggestion
let myFunction = () => console.log("Function was invoked!");
```